packages/terra-image/tests/jest/DecorativeImage.test.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ThemeContextProvider from 'terra-theme-context/lib/ThemeContextProvider';
+
+import DecorativeImage from '../../src/variants/_DecorativeImage';
+
+/* Decorative Image component */
+it('should render a default component with empty attribute', () => {
```

Comment:
This doc test string is unclear. What is being tested here?

Reply:
I missed the `alt` here. [updated](https://github.com/cerner/terra-core/pull/3585/commits/a72252e81ed3e48fbebc3ff03878cf6f428b0261#diff-d7472c378e1cf7e5f9153f7e4e7b7ce36ef94d78c3851da233358a07afe02cff) !!